src/Microsoft.Data.Sqlite/Interop/NativeMethods.cs

```diff
@@ -6,42 +6,11 @@
 using System.Runtime.InteropServices;
 using static Microsoft.Data.Sqlite.Interop.Constants;
 
-#if NET451 || DOTNET5_4
-using System.IO;
-using System.Reflection;
-using Microsoft.Extensions.PlatformAbstractions;
-using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Internal;
-#endif
 
 namespace Microsoft.Data.Sqlite.Interop
 {
     internal static class NativeMethods
     {
-#if NET451 || DOTNET5_4
-        static NativeMethods()
-        {
-            var loaded = NativeLibraryLoader.TryLoad("sqlite3");
```

Comment:
Wait, we still need this on Full .NET! Only UWP and DNX will handle the runtimes stuff.